governance-release-process/process.md

```diff
@@ -26,8 +26,8 @@ If the change proposal is endorsed by the INSPIRE CT or by the MIG-T, respective
 
 If no objection is raised by MIG members during the two-week time window, or if the change proposal is endorsed by the MIG during the meeting, then:
 
-1) If the endorsed change proposal has an _impact on the IR_, the INSPIRE CT shall submit a proposal for a draft amendment of the IR. The endorsement of this proposal is subject to its own governance process (involving the Comitology procedure) and is outside the scope of the present document.
-2) If the endorsed change proposal has no impact on IR, the JRC shall implement the change in the relevant schema and in the UML (if it is impacted). Once this is done, the issue is finally closed. If the endorsed change proposal has (also) an _impact on TG_, the related changes will follow the specific governance process (defined in the [Technical Guidance documents repository](https://github.com/INSPIRE-MIF/technical-guidelines)), based on a specific group of actors (similar to the one for schemas) and a different release plan. If the endorsed change proposal has (also) an _impact on the validator_, an issue shall be raised in the [validator helpdesk](https://github.com/INSPIRE-MIF/helpdesk-validator).
+1) If the endorsed change proposal has no impact on IR, the JRC shall implement the change in the relevant schema and in the UML (if it is impacted). Once this is done, the issue is finally closed. If the endorsed change proposal has (also) an _impact on TG_, the related changes will follow the specific governance process (defined in the [Technical Guidance documents repository](https://github.com/INSPIRE-MIF/technical-guidelines)), based on a specific group of actors (similar to the one for schemas) and a different release plan. If the endorsed change proposal has (also) an _impact on the validator_, an issue shall be raised in the [validator helpdesk](https://github.com/INSPIRE-MIF/helpdesk-validator).
+2) If the endorsed change proposal has an _impact on the IR_, the INSPIRE CT shall submit a proposal for a draft amendment of the IR. The endorsement of this proposal is subject to its own governance process (involving the Comitology procedure) and is outside the scope of the present document. If the change proposal will be endorsed and after the publication of the revised IR, the JRC shall implement the change as specified above.
```

Comment:
just a comment to try to include, in the text, the new label "IR revised". this would 100% link the text to the graphics and make things fully clear.

Reply:
Thanks, suggestions have been incorporated. The same changes have been introduced also in the TG workflow.